install/src/command.rs

```diff
@@ -1283,3 +1283,29 @@ pub fn run(
         }
     }
 }
+
+pub fn list(config_file: &str) -> Result<(), String> {
+    let config = Config::load(config_file)?;
+
+    let entries = fs::read_dir(&config.releases_dir);
+    if entries.is_err() {
+        return Err("please make sure your config is set up properly".to_string());
+    }
+
+    for entry in entries.unwrap() {
+        let entry_path = entry.unwrap().path();
+        let dir_name = entry_path.file_name().unwrap().to_string_lossy();
```

Comment:
In general, explicit `unwrap()` should be avoided. A couple preferred options are using the `?` operator and/or `.map()` to only perform logic on `Ok(...)`'s

Reply:
Thanks for the suggestion.

We can leave them as they are since, we can catch invalid directory entries using the change you suggested above. 

```suggestion
    let entries = fs::read_dir(&config.releases_dir).map_err(|err| format!("Failed to read install directory, double check that your configuration file is correct: {err}"))?;
```